CHANGELOG.md

```diff
@@ -28,7 +28,12 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Deprecations
 
--
+- Specifying "Install into all namespaces in the cluster" or otherwise setting
+  the `WATCH_NAMESPACES` environment variable has no effect anymore. This was
+  done to improve operator's memory usage as reported in
+  [an upstream issue](https://github.com/ComplianceAsCode/compliance-operator/issues/40)
+  All Compliance Operator's CRDs must be installed in operator's own namespace
```

Comment:
I think here you mean `CRs`

Reply:
Other than this, looks good!

Reply:
oops, yes. Thanks for the review, fixed!